project-templates/feature_request.md

```diff
@@ -0,0 +1,45 @@
+---
+name: Feature request
+about: Suggest new feature
+title: "feat: "
+labels: ''
+assignees: ''
+
+---
+
+### Title
+
+*Example: Create Members GET API*
+
+### Description
+
+Description of the new feature and the problem it is solving.
```

Comment:
Just a suggestion (with more explanation of what to expect in this part).

```
Provide description of a new feature. Include:  
- How it should work.  
- Any specific requirements or edge cases.  
- Mockups or examples, if applicable.
```

And I like the example 👍